charts/managed-serviceaccount/templates/clusterrole.yaml

```diff
@@ -37,6 +37,9 @@ rules:
       - watch
       - update
       - patch
+      {{- if (.Values.featureGates | default dict).ephemeralIdentity }}
```

Comment:
some how `and .Values.featureGates .Values.featureGates. ephemeralIdentity ` doesn't work for handling nil pointer, so used this trick 😂